aws/cloudformation/rdp_bastion.yaml

```
@@ -0,0 +1,501 @@
+---
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  This template creates a bastion host based on Mate AMI on a new VPC
+  (customer).
+  This bastion host will be used to connect to the ROSA cluster, run oc commands
+  and view the OpenShift console.
+
+Parameters:
+  pTagprefix:
+    Description: Use the course code to prepend resource tags
+    Type: String
+    Default: cs220
+  pCustomerVpcCidrBlock:
+    AllowedPattern: '((\d{1,3})\.){3}\d{1,3}/\d{1,2}'
+    Default: 10.2.0.0/16
+    Description: Customer VPC CIDR block (10.2.0.0/16 for example).
+    Type: String
+  pCustomerVpcPrivateSubnetCidrBlock:
+    AllowedPattern: '((\d{1,3})\.){3}\d{1,3}/\d{1,2}'
+    Default: 10.2.0.0/24
+    Description: >
+      CIDR block for the private subnet in the customer VPC
+      (10.2.1.0/24 for example).
+    Type: String
+  pCustomerVpcPublicSubnetACidrBlock:
+    AllowedPattern: '((\d{1,3})\.){3}\d{1,3}/\d{1,2}'
+    Default: 10.2.2.0/24
+    Description: >
+      CIDR block for the public subnet in the customer VPC
+      (10.2.2.0/24 for example).
+    Type: String
+  pInstanceType:
+    Description: EC2 instance type
+    Type: String
+    Default: "t3.micro"
+  pSshLocation:
+    Description: The IP address range that can SSH to the EC2 instance
+    Type: String
+    Default: "0.0.0.0/0"
+  pRdpLocation:
+    Description: The IP address range that can use RDP to the EC2 instance
+    Type: String
+    Default: "0.0.0.0/0"
+
+Mappings:
+  RegionMap:
+    ap-south-1:
+      ami: ami-061183ad486d5dd8a
+    eu-north-1:
+      ami: ami-03a2ff446d5bf5187
+    eu-west-3:
+      ami: ami-06e90f3404ebc4277
+    eu-west-2:
+      ami: ami-0ef262972e641bb3e
+    eu-west-1:
+      ami: ami-0ac03fb170870f7c7
+    ap-northeast-3:
+      ami: ami-01d7109b399c6b223
+    ap-northeast-2:
+      ami: ami-06b9122710049dfe7
+    ap-northeast-1:
+      ami: ami-0d5142f63c808d143
+    ca-central-1:
+      ami: ami-04609b5b156500b17
+    sa-east-1:
+      ami: ami-0b123273fd25fe833
+    ap-southeast-1:
+      ami: ami-0fcd2e9ac9a168217
+    ap-southeast-2:
+      ami: ami-09fb3a51968a858cd
+    eu-central-1:
+      ami: ami-0087bd2f5c26af5ca
+    us-east-1:
+      ami: ami-005b11f8b84489615
+    us-east-2:
+      ami: ami-08778753ef37aa408
+    us-west-1:
+      ami: ami-0267fc24ee0102728
+    us-west-2:
+      ami: ami-081aaface2871d0d0
+
+Resources:
+  # Customer VPC
+  rCustomerVpc:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock:
+        Ref: pCustomerVpcCidrBlock
+      EnableDnsSupport: true
+      EnableDnsHostnames: true
+      Tags:
+        - Key: Name
+          Value: !Join ["", [!Ref pTagprefix, "-customer-vpc"]]
```

```suggestion
          Value: !Join ["", [!Ref pTagprefix, "-customer-vpc-bastion"]]
```

Review the changes from our conversation regarding bastion/vpn